index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="equalizer.css" /> 
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="style.css" /> 
+  </head>
+  <body>
+
+    <div class="orange-margin"> </div>
```

Comment:
this shouldn't be a div, use border top in the `main-images` div. I would change that class name to `main-section`  so it makes more sense when you read it